modules/BuildTools.Tasks/CreateSourceLink.cs

```diff
@@ -51,20 +51,52 @@ public override bool Execute()
             SourceLinkRoot += '*';
 
             var codeSource = ConvertUrl();
-            var data = new JObject
+
+            // TODO allow multiple. See https://github.com/aspnet/Universe/issues/858
```

Comment:
Stale comment?

Reply:
I cherry picked this file completely without changes from release/2.2. But yes, this comment appears to be stale.